<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>investClass</class>
 <widget class="QMainWindow" name="investClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>1266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>invest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ButtonMakeMidi">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>500</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>01フロップデザイン</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>MakeMidi!</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelMidiName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>01フロップデザイン</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存名を入力</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonChoseVideo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>01フロップデザイン</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ビデオを選択</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelVideoName">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>800</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>01フロップデザイン</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonCancel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>1020</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>01フロップデザイン</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextMidiName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextEditorInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelImageLeft">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>580</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelProcess">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>540</y>
      <width>311</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>01フロップデザイン</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Processing...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>220</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>01フロップデザイン</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Piano</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Organ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Harmonica</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Guitar</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ButtonChoseSave">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>01フロップデザイン</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存先を選択</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelSaveName">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>800</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>01フロップデザイン</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>10</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSub">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>01フロップデザイン</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Trumpet</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sax</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Flute</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Recorder</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="LabelInst_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>01フロップデザイン</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>メインの楽器</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelInst_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>01フロップデザイン</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>サブの楽器</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonEnd">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>1120</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>01フロップデザイン</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>終了</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAiueo">
   <property name="text">
    <string>aiueo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="invest.qrc"/>
 </resources>
 <connections/>
</ui>
